zephyr/boards/arm/ubx_mininorab12_nrf5340/ubx_mininorab12_nrf5340_cpunet-pinctrl.dtsi

```diff
@@ -0,0 +1,63 @@
+/*
+ * u-blox MINI-NORA-B12 board configuration
+ *
+ * Copyright (c) 2023 u-blox AG
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&pinctrl {
+	uart0_default: uart0_default {
+		group1 {
```

Comment:
Pls double check pin numbers, not sure if I understand how to define this.

Reply:
Followed nrf5340dk_nrf5340 style. Pins were swapped, corrected now.